include/MeshcatCpp/Material.h

```diff
@@ -13,6 +13,7 @@
 #include <optional>
 #include <unordered_map>
 #include <utility>
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```

In C++ the correct header is `cstdint`, `stdint.h` is the C-only version.